js/components/Toolbox.js

```diff
@@ -0,0 +1,28 @@
+const React = require("react");
```

Comment:
The name `toolbox` is somewhat of a placeholder because that's a fairly well defined word. Do you think it works here?